conf/messages.en

```diff
@@ -463,3 +463,10 @@ item.transportEquipment.placeholder = Select a transport equipment
 item.transportEquipment.change.hidden = transport equipment for item {0}
 item.transportEquipment.checkYourAnswersLabel = Transport equipment
 item.transportEquipment.error.required = Select which transport equipment you are using for this item
+
+item.addTransportChargesYesNo.title = Do you want to add a method of payment for this item's transport charges?
+item.addTransportChargesYesNo.heading = Do you want to add a method of payment for this item's transport charges?
+item.addTransportChargesYesNo.paragraph = These include charges for hauliers and carriers.
+item.addTransportChargesYesNo.checkYourAnswersLabel = Do you want to add a method of payment for this item's transport charges?
+item.addTransportChargesYesNo.hidden = if you want to add a method of payment for this item's transport charges
+item.addTransportChargesYesNo.error.required = Select yes if you want to add a method of payment for this item's transport charges
```

Comment:
Incorrect apostrophes being used. Should be `’` instead of `'`'

Reply:
Done